internal/controller/virtualmachinebmc/controller.go

```diff
@@ -256,8 +254,8 @@ func (r *VirtualMachineBMCReconciler) validateVirtualMachineExists(ctx context.C
 //+kubebuilder:rbac:groups=bmc.kubevirt.io,resources=virtualmachinebmcs/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=bmc.kubevirt.io,resources=virtualmachinebmcs/finalizers,verbs=update
 //+kubebuilder:rbac:groups="",resources=pods;services;serviceaccounts,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=rbac.authorization.k8s.io,resources=clusterroles,verbs=bind
```

Comment:
This is required for the controller manager to create RoleBindings for agents. The referenced ClusterRole contains permissions with CDI DataVolume, which the controller manager's ClusterRole does not possess.